Docs/tutorials

davidhunter08 opened the pull request:
Installation instructions and tutorials have been added to the docs.

visualunderground commented on app/views/examples/postcode-search.js:
```diff
@@ -0,0 +1,90 @@
+}
+
+function isPostcodeFormatValid (postcode) {
+  return /^([A-Z][A-Z0-9]?[A-Z0-9]?[A-Z0-9]? {1,2}[0-9][A-Z0-9]{2})$/i.test(postcode)
```
Looks like the space is mandatory. getAddress.io doesn't impose this, can we make it optional?

paul-mitchell-bjss replied:
Agreed. Good spot.

davidhunter08 committed:
Nunjucks macro tutorial page

Added content to 'how to use nunjucks macros' page.
Updated docs home link on the breadcrumb to go to `/docs`.

davidhunter08 committed:
Updated content on writing css docs page.

davidhunter08 committed:
Updated nunjucks macro documentation.

davidhunter08 committed:
Writing CSS page content.

davidhunter08 committed:
Removed test content from index.

davidhunter08 committed:
Branching example and install docs

Updated the question on the branching example page.

Added the stages of the simple install guide to the install page.

davidhunter08 committed:
Fixed breadcrumb home link on about page.

davidhunter08 committed:
Branching example and install docs.

Linked the branching example from the tutorials page.

Updated content on install docs.

davidhunter08 committed:
Nunjucks tutorial page content and install breadcrumb home link

davidhunter08 committed:
Updated the radios name on the branching example.

davidhunter08 committed:
Updated content on maing pages and using nunjucks docs.

davidhunter08 committed:
Amended nunjucks macro page content.

davidhunter08 committed:
Prototype kit navigation - header and footer links

davidhunter08 committed:
Started content for about page.

davidhunter08 committed:
Installation, tutorials and about docs.

Updated:
- intro on the index and about pages
- checking node.js instructions on requirements page
- tutorial links on advanced install page

davidhunter08 committed:
Removed lede text from install page.

davidhunter08 committed:
Fixed broken link on install guide.

davidhunter08 committed:
Installation docs.

Added new screenshots of folder structure and welcome page.
